lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -137,12 +137,13 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
             system_identifier = INTERNAL_SYSTEM)
 
     if not patient_entry:
-        LOG.warning("Skipping record with insufficient information to construct patient")
+        LOG.warning("Skipping record with insufficient information to construct patient for record: "
+            f"{enrollment.get('record_id')}")
```

Comment:
The same convention could be applied here, like: 
"Skipping record {enrollment.get('record_id')} with insufficient information to construct patient"

Reply:
Changed!